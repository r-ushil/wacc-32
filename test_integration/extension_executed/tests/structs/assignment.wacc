begin
  struct IntBox {
    int x
  }
  IntBox box = IntBox { x: 1 };
  println box.x;
  box.x = 42;
  println box.x
end