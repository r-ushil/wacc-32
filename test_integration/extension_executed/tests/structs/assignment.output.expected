Compiler exit code: 0
--------- stdout ---------
1
42
--------------------------
Exit code: 0
